bittensor/axon.py

```diff
@@ -194,6 +194,7 @@ def _forward(self, request):
                 code: (:obj:`bittensor_pb2.ReturnCode, `required`)
                     return code associated with forward call i.e. Success of Timeout.
         """
+        device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
this should be pulled directly from the synapse, not from this line.

ie. synapse.device?

Reply:
Fixed